analysis/tc_tackle_testapp_private.go

```diff
@@ -0,0 +1,29 @@
+package analysis
+
+import (
+	"github.com/konveyor/go-konveyor-tests/data"
+	"github.com/konveyor/go-konveyor-tests/data/identity"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var TackleTestappPrivateBinary = TC{
+	Name:        "tackle-testapp-binary",
+	Application: data.TackleTestapp,
+	Identities: []api.Identity{
+		identity.TackleTestappMaven,
+	},
+	Task: Analyze,
+	Binary: true,
+	Analysis: api.Analysis{
+		Effort: 16,
+		Issues: []api.Issue{
+			{Category: "mandatory", Description: "Java native libraries (JNI, JNA)", Effort: 7, Rule: "jni-native-code-00000"},
+			{Category: "mandatory", Description: "File system - Java IO", Effort: 1, Rule: "local-storage-00001"},
+			{Category: "potential", Description: "Java Servlet", Effort: 0, Rule: "javaee-technology-usage-00120"},
```

Comment:
@aufi @pranavgaikwad  In the post analysis check - it ignores non-mandatory issues from analysis report so this test fails on `analysis_test.go:175: Different amount of issues error. Got 2, expected 3.`
Should I remove the potential issue?

Reply:
The correctly working example is https://github.com/konveyor/go-konveyor-tests/blob/main/analysis/tc_tackle_testapp_public_deps.go  (with deps and updated target), so right, there should be just 2.

Reply:
Done in e10a7d0